### PR TITLE
docs: misc improved docs, including rust watcher example

### PR DESCRIPTION
**Pull request by brayo-pip**
| :rocket: | This description was created by [Ellipsis](https://www.ellipsis.dev) for commit 27903cb82a6fccf2030155d55e512480f8b1b681  | 
|--------|--------|

### Summary:
Updated documentation with new Rust examples, fixed Python example, and added configuration details for `aw-server-rust`.

**Key points**:
- Updated `src/configuration.rst` to include `host`, `port`, and `cors` options for `aw-server-rust`.
- Added `src/examples/cargo.toml` for Rust client dependencies.
- Modified `src/examples/client.py` to fix `create_bucket` function call.
- Added `src/examples/client.rs` with a detailed Rust client example.
- Added `src/examples/minimal_client.rs` with a minimal Rust client example.
- Updated `src/examples/writing-watchers.rst` to include Rust examples for writing watchers.
- Removed outdated LastFM importer reference from `src/importers.rst`.
- Added `aw-watcher-lastfm` to media watchers in `src/watchers.rst`.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

File: src/examples/writing-watchers.rst [hunk 1]
```diff
@@ -34,3 +34,38 @@ This example will describe how to:
 
 .. literalinclude:: client.py
 
+Writing your first watcher in rust
+==================================
+
+To get started with writing watchers in Rust, you need to add the `aw-client-rust` and `aw-model` crates to your `Cargo.toml` file.
+The most upto date versions depend directly on :gh-aw:`aw-server-rust`.
```
Comment (ErikBjare) [inline src/examples/writing-watchers.rst:41]:
```suggestion
The most up-to-date versions depend directly on :gh-aw:`aw-server-rust`.
```

File: src/examples/writing-watchers.rst [hunk 2]
```diff
@@ -34,3 +34,38 @@ This example will describe how to:
 
 .. literalinclude:: client.py
 
+Writing your first watcher in rust
+==================================
+
+To get started with writing watchers in Rust, you need to add the `aw-client-rust` and `aw-model` crates to your `Cargo.toml` file.
```
Comment (ErikBjare) [inline src/examples/writing-watchers.rst:40]:
```suggestion
To get started with writing watchers in Rust, you need to add the ``aw-client-rust`` and ``aw-model`` crates to your ``Cargo.toml`` file.
```

File: src/configuration.rst [hunk 3]
```diff
@@ -20,7 +20,9 @@ aw-server-python
 aw-server-rust
 --------------
 
-TODO
+- :code:`host` Hostname to start the server on. Currently only :code:`localhost` or :code:`127.0.0.1` are supported.
+- :code:`port` Port number to start the server on.
+- :code:`cors` Enable CORS (Cross-Origin Resource Sharing) for the server. This is useful if you want to access the server from a different domain than the one it's hosted on.
```
Comment (ErikBjare) [inline src/configuration.rst:25]:
```suggestion
- :code:`cors` List of allowed CORS domains (Cross-Origin Resource Sharing) for the server. Useful in testing and development to let other web origins access the ActivityWatch API, such as aw-webui in development mode on port 27180.
```

File: src/examples/writing-watchers.rst [hunk 4]
```diff
@@ -34,3 +34,38 @@ This example will describe how to:
 
 .. literalinclude:: client.py
 
+Writing your first watcher in rust
```
Comment (ErikBjare) [inline src/examples/writing-watchers.rst:37]:
```suggestion
Writing your first watcher in Rust
```

File: src/examples/writing-watchers.rst [hunk 5]
```diff
@@ -1,5 +1,5 @@
-Writing your first watcher
-==========================
+Writing your first watcher in python
```
Comment (ErikBjare) [inline src/examples/writing-watchers.rst:1]:
```suggestion
Writing your first watcher in Python
```

**Review by ErikBjare — APPROVED** [review]
Nice! Applying some changes and merging.

**Comment by brayo-pip** [comment]
@ErikBjare the syntax highlighting is not working for the rust examples, I was going to investigate it later. But I can include it in the next PR.

**Pull request merged**